src/views/SystemProfile/index.scss

```diff
@@ -0,0 +1,55 @@
+#main-content {
+    .system-profile {
+        &__summary-box {
+            border-top: 0;
+            background-color: color($theme-color-primary-lighter);
+            padding-left: 0;
+            padding-right: 0;
+        }
+
+        &__acronym {
+            font-size: 1rem;
+            font-weight: normal;
+        }
+
+        &__summary-body {
+            font-weight: normal;
+            font-size: 1rem;
+        }
+
+        &__subheader {
+            font-size: 1.5rem;
+        }
+
+        &__nav {
+            padding: 0;
+        }
+
+        &__nav-button {
+            width: 100%
+        }
+    }    
+
+    .usa-breadcrumb {
+        background-color: transparent;
+        padding: 0;
```

Comment:
`padding: 0` causes the breadcrumb text to bump right up against the navigation bar:
![breadcrumbs - no padding](https://user-images.githubusercontent.com/92891039/151060530-1290a5b8-2fdc-45ef-b8b5-c48341d7f0ae.PNG)

Contrast the version with padding that's currently deployed:
![breadcrumbs - with padding](https://user-images.githubusercontent.com/92891039/151060589-b8aad478-d127-4893-b089-746b1799537a.PNG)

What better matches the wireframe/designs?

Reply:
Nice catch you are right, I though the mod to usa-breadcrumb was nested under the system-profile class.  Will update

Reply:
Looks better with https://github.com/CMSgov/easi-app/pull/1437/commits/7ef92399064e1f01b41071aff2398f6b6b4d0b12.